### ISSUE TITLE
cannot install with current version of Psalm (v4.1)

### ISSUE THREAD

**Issue by dwenaus** [humanmade/psalm-plugin-wordpress#6]
it gives a composer error

```
  Problem 1
    - humanmade/psalm-plugin-wordpress 1.0.1 requires vimeo/psalm ^3.11 -> satisfiable by vimeo/psalm[3.11.0, 3.11.1, 3.11.2, 3.11.3, 3.11.4, 3.11.5, 3.11.6, 3.11.7, 3.12.0, 3.12.1, 3.12.2, 3.13.0, 3.13.1, 3.14.0, 3.14.1, 3.14.2, 3.15, 3.16, 3.17.0, 3.17.1, 3.17.2, 3.18.0, 3.18.2, 3.x-dev] but these conflict with your requirements or minimum-stability.
    - humanmade/psalm-plugin-wordpress 1.0.0 requires vimeo/psalm ^3.11 -> satisfiable by vimeo/psalm[3.11.0, 3.11.1, 3.11.2, 3.11.3, 3.11.4, 3.11.5, 3.11.6, 3.11.7, 3.12.0, 3.12.1, 3.12.2, 3.13.0, 3.13.1, 3.14.0, 3.14.1, 3.14.2, 3.15, 3.16, 3.17.0, 3.17.1, 3.17.2, 3.18.0, 3.18.2, 3.x-dev] but these conflict with your requirements or minimum-stability.
    - Installation request for humanmade/psalm-plugin-wordpress ^1.0 -> satisfiable by humanmade/psalm-plugin-wordpress[1.0.0, 1.0.1].
```

my composer entry looks like this 
```
        "vimeo/psalm": "^4.1",
```

Why not just remove the requirement for Psalm because this plugin does not actually require Psalm. A fork of this repo has done just that. (I think they are using the stub files for PHPStan.)

**Comment by joehoyle** [comment]
I've updated the requirement to 4.1. It might be a good idea to remove it all together, though I'm not sure. 4.x Psalm broke the plugin combat with 3.x plugins, so I think the version requirement is good for making that explicit.

**Comment by joehoyle** [comment]
FYI I released 2.0.0 in https://github.com/humanmade/psalm-plugin-wordpress/releases/tag/2.0.0 which includes this update.
